src/propmcwf.jl

```diff
@@ -0,0 +1,90 @@
+immutable QuMCWF <: QuPropagatorMethod
+end
+
+type QuMCWFEnsemble{QA<:QuBase.AbstractQuArray}
+    ntraj
+    rho::QA
+    decomp
+    QuMCWFEnsemble(ntraj, rho, decomp) = new(ntraj, rho, decomp)
+end
+
+QuMCWFEnsemble{QM<:QuBase.AbstractQuMatrix}(rho::QM) = QuMCWFEnsemble{QM}(500, rho, eigfact(reshape(full(coeffs(rho)), size(rho))))
```

Comment:
I think you don't need the reshape here.

And I would say one always wants to pass `ntraj` (maybe with a default value) as well.